README.md

```diff
@@ -21,14 +21,26 @@ sudo make install
 make clean
 ```
 
+### Uninstalling 
+
+```
+cd /path/to/xwinwrap
+make
```

Comment:
why is this here?

Reply:
If you refer to line 27, I use the "cd" command, because when the user opens for example a new terminal he must first go to the installation directory to uninstall the program.